src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.spec.ts

```diff
@@ -71,4 +97,19 @@ describe('DotEditPageMainComponent', () => {
         const nav: DotEditPageNavComponent = fixture.debugElement.query(By.css('dot-edit-page-nav')).componentInstance;
         expect(nav.pageState).toEqual(mockDotRenderedPageState);
     });
+
+    it('should call the ContentletEditorService Edit when OpenProperties evt happens', () => {
```

Comment:
You're not testing here that the event is coming from the nav bar, if we remove the binding in the nav bar this test will not break but the UI will.

You can use `triggerEventHandler` to fake trigger event from any component.